core/services/ocrbootstrap/delegate.go

```diff
@@ -32,6 +33,13 @@ type Delegate struct {
 	isNewlyCreatedJob bool
 }
 
+// Extra fields to enable router proxy contract support. Must match field names of functions' PluginConfig.
+type relayConfigRouterContractFields struct {
+	DONId                           string `json:"donId"`
```

Comment:
`DONID`? That's awkward though... :thinking: 
```suggestion
	DONID                           string `json:"donId"`
```

Reply:
Sure.